### DISCUSSION TITLE
SSL Protocol error

### DISCUSSION THREAD

**Discussion by itslevir** [supabase/supabase#2348] [Questions] [answered]
I created a supabase project about 15-20 minutes ago and the tables in the database were never loading. I look to the console for answers, and find these errors:
![image](https://user-images.githubusercontent.com/72626657/125521447-acb045e4-5a3c-4f17-84fc-f7e3357b254f.png)
Is this an ongoing issue? I've tried switching browsers, too, to no avail.

**Comment by inian** [comment, marked as answer]
Hi @lerichardson, replied in the issue you created

**Reply by itslevir** [reply]
Thank you!
